src/org/usfirst/frc/team4915/steamworks/subsystems/Launcher.java

```diff
@@ -174,7 +174,7 @@ private boolean isLauncherAtSpeed()
         final double epsilon = 100; // allow 100 RPM of error.
         double speedTarget = SmartDashboard.getNumber("Launcher_TGT", Launcher.DEFAULT_LAUNCHER_SPEED);
         double speedActual = m_launcherMotor.getSpeed();
-        if (speedActual >= speedTarget - epsilon || speedActual <= speedTarget + epsilon)
+        if (speedActual >= speedTarget - epsilon && speedActual <= speedTarget + epsilon)
```

Comment:
Better. I suggest also using more parenthesis around the expressions to make intent more clear (although it is correct as shown).
For example "if ((speedActual >= (speedTarget - epsilon)) && (speedActual <= (speedTarget + epsilon)))"